src/pages/index.js

```diff
@@ -37,15 +37,15 @@ const styles = theme => ({
             </Avatar>
           }
           action={
-            <>
-              <Button
-                variant="contained"
-                color="secondary"
-                className={props.classes.root}
-              >
-                <Link to="/products">View All Products</Link>
-              </Button>
-            </>
+            <Button
```

Comment:
This is not related to the change, but nesting Link under Button results in '<a>' tag nested inside of a '<button>' tag.